serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc

```diff
@@ -1 +1,194 @@
-//= Mocking HTTP CloudEvents sink with Wiremock
\ No newline at end of file
+= Mocking HTTP CloudEvents Sink with Wiremock
+==========================================================
+:compat-mode!:
+// Metadata:
+:description: Mocking HTTP CloudEvents sink with Wiremock
+:keywords: kogito, workflow, quarkus, serverless, test, integration, wiremock, CloudEvents
+// Referenced documentation pages.
+:basic_integration_test_with_restassured_guide: xref:testing-and-troubleshooting/basic-integration-tests-with-restassured.adoc
```

Comment:
maybe append the "_url" suffix to these envs?

Reply:
I just appended the "_guide" for our guides references regarding is clear the rest of them are urls. I think is clear enough where the env vars point, but I can rename the external ones with the sufix "_url" even I think they are clear enough, and grouped in header making them easy to maintain. @ricardozanini,  WDYT?

Reply:
I think we should take the same approach for every guide to have a clear pattern. IIRC we are using `_url` suffix for any URL variable reference, so we should stick to it. If we change, we need to change all.

Reply:
@spolti @ricardozanini I agree, Thanks both!
I've added the `_url` to the external references variables.